test/helpers.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+import (
+	"fmt"
+	"strings"
+)
+
+func loadMockJson(filename string) ([]byte, error) {
+	// TODO EJR where should the mockjsons be stored? or should the feature file just hold the full path somehow
+	return nil, nil
+}
```

Comment:
@winder I wonder where we should store the mock jsons to pass in to the test, and how best to generate the mock jsons

Reply:
put them in the `algorand-sdk-testing` repo. The next logical step for that repo is probably embedding it as a sub-repo of each SDK. That way we can assume the files exist in a relative directory.